.github/workflows/test.yml.jinja

```diff
@@ -0,0 +1,65 @@
+name: test Odoo addons
+
+on:
+  pull_request:
+    branches:
+      - "{{ odoo_version }}*"
+  push:
+    branches:
+      - "{{ odoo_version }}*"
+
+{%
+set IMAGES = {
+  "odoo": {
+    "14.0": "ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest",
+    "13.0": "ghcr.io/oca/oca-ci/py3.6-odoo13.0:latest",
+    "12.0": "ghcr.io/oca/oca-ci/py3.6-odoo12.0:latest",
+    "11.0": "ghcr.io/oca/oca-ci/py3.5-odoo11.0:latest",
+    "10.0": "ghcr.io/oca/oca-ci/py2.7-odoo10.0:latest",
+  },
+  "ocb": {
+    "14.0": "ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest",
+    "13.0": "ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest",
+    "12.0": "ghcr.io/oca/oca-ci/py3.6-ocb12.0:latest",
+    "11.0": "ghcr.io/oca/oca-ci/py3.5-ocb11.0:latest",
+    "10.0": "ghcr.io/oca/oca-ci/py2.7-ocb10.0:latest",
+  }
+}
+-%}
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: {{ "${{" }} matrix.container {{ "}}" }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - container: {{ IMAGES["odoo"][odoo_version] }}
+            makepot: "true"
+          - container: {{ IMAGES["ocb"][odoo_version] }}
+    services:
+      postgres:
+        image: postgres:9.6
```

Comment:
I think we should use more modern postgres versions.

At least, I'd use the latest stable for the date of release of each odoo version. You can get the relevant info from https://www.postgresql.org/support/versioning/

Reply:
My position about this is that we should test with the lowest python and postgres version that Odoo supports, and rely on python and postgres to assure backward compatibility in more modern versions. But we can certainly discuss this with the wider community.

Reply:
Well, OK. I guess the best would be to just do a matrix with all supported versions, but that has another kind of costs and implications, and in any case it seems outside this PR scope, so we can move forward.